tests/mocks/handlers/collections/prompt.ts

```diff
@@ -0,0 +1,30 @@
+import { fileURLToPath } from "url";
+import filenamify from "filenamify";
+import fs from "fs";
+import path from "path";
+import { http, HttpResponse } from "msw";
+
+const COLLECTIONS_DATA_DIR = path.resolve(
+  fileURLToPath(import.meta.url),
+  "../../../data/collections"
+);
+
+// note that mock test path becomes works/work_id/chapter_id.html
```

Comment:
Could you explain this a bit better? I'm not sure where the `/chapters/` directory is in the URL. Unless this is a cop/paste from another file in which case you can get rid of it.